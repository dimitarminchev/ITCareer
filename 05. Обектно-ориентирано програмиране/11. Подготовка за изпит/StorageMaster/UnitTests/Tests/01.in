RegisterStorage DistributionCenter SofiaDistribution
RegisterStorage Warehouse AmazonWarehouse
AddProduct Gpu 1200
AddProduct SolidStateDrive 205
AddProduct HardDrive 70
AddProduct HardDrive 120
SelectVehicle SofiaDistribution 0
LoadVehicle HardDrive Gpu
SendVehicleTo SofiaDistribution 0 AmazonWarehouse
UnloadVehicle AmazonWarehouse 3
END
