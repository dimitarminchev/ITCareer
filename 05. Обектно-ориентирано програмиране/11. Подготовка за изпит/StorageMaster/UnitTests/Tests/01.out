Registered SofiaDistribution
Registered AmazonWarehouse
Added Gpu to pool
Added SolidStateDrive to pool
Added HardDrive to pool
Added HardDrive to pool
Selected Van
Loaded 2/2 products into Van
Sent Van to AmazonWarehouse (slot 3)
Unloaded 2/2 products at AmazonWarehouse
AmazonWarehouse:
Storage worth: $1320.00
SofiaDistribution:
Storage worth: $0.00
